src/pages/FlssPrototypes/FlssLoanRow.vue

```diff
@@ -75,19 +75,22 @@
 </template>
 
 <script>
-// import basicLoanQuery from '@/graphql/query/basicLoanData.graphql';
-// import basicLoanQuery from '@/graphql/query/recommendedLoans.graphql';
+
 // import loanChannel from '@/graphl/query/loanChannelData.graphql'
+// import KvDropdown from '@/components/Kv/KvDropdown';
 import KvCarousel from '@/components/Kv/KvCarousel';
 import KvCarouselSlide from '@/components/Kv/KvCarouselSlide';
 import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
 import LoanCardController from '@/components/LoanCards/LoanCardController';
+// import flssLoanQuery from '@/graphql/query/flssQuery.graphql';
 import basicLoanQuery from '@/graphql/query/flssQuery.graphql';
```

Comment:
For the next stage of enhancement you'll want to rename these queries to match what their graphql file contains. `flssLoanQuery` in this case.